packages/carbon-graphs/src/js/controls/Graph/Graph.js

```diff
@@ -422,68 +422,129 @@ class Graph extends Construct {
   /**
      * Updates the graph axisData and content.
      *
-     * @param {Array} graphData - Input array that holds updated values, key and labels
+     * @param {Object} graphData - Input object that holds updated values and key pair and labels
      * @returns {Graph} - Graph instance
      */
   reflow(graphData) {
-    let position;
+    console.warn('reflow is now deprecated and will be removed in future Major Version Bump. Please use reflowMultipleDatasets');
```

Comment:
nit: reflow is deprecated and will be removed a future major release. Please use reflowMultipleDatasets instead.